npc/warps/cities/prontera.txt

```diff
@@ -3,46 +3,48 @@
 //===== By: ==================================================
 //= Athena (1.0)
 //= Nana (1.2)
+//= Tranquility (1.6)
 //===== Current Version: =====================================
-//= 1.5
+//= 1.6
 //===== Description: =========================================
 //= Warp Points for Prontera
 //===== Additional Comments: =================================
 //= 1.3 prt19 warp fix [shadow]
 //= 1.4 Added warps for the sign quest [MasterOfMuppets]
 //= 1.5 Corrected NPC names to fall within proper restrictions. [L0ne_W0lf]
+//= 1.6 Corrected Warps for New Prontera. [Tranquility]
 //============================================================
 
 //= Prontera City ============================================
-prontera,107,215,0	warp	prt01	2,2,prt_in,240,139
+prontera,105,214,0	warp	prt01	2,2,prt_in,240,139
```

Comment:
This is a bit controversial I guess.

Yes, I believe not many people use the new prontera map, so the old warps must stay around. It can be either another file (or at least, a commented out part in the current file, but that makes it harder to toggle)

Considering that we still ship the mapcache for the old version, the old warps should be enabled by default.